pkg/apis/autoscaling/v1/conversion.go

```diff
@@ -449,6 +450,20 @@ func Convert_v1_HorizontalPodAutoscaler_To_autoscaling_HorizontalPodAutoscaler(i
 	// drop round-tripping annotations after converting to internal
 	out.Annotations, _ = autoscaling.DropRoundTripHorizontalPodAutoscalerAnnotations(out.Annotations)
 
+	// Until kube 1.27 we're still storing autoscaling as v1, but the HPA controller is consuming it as v2. Behaviors are an annotation in v1 and can be partially
```

Comment:
what does this do to the annotations in question in a flow like
1. write object to API/etcd, scaleup gets set
2. read object as v1, do annotations make sense?